### PR TITLE
Update Harbor

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
harbor

**Issue:** contributes toward #9072 

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from https://github.com/cncf/artwork/blob/ff2b2b5216e22f001ddd444ca580c484dd10302e/projects/harbor/icon/black/harbor-icon-black.svg
Color: Unchanged

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5776,7 +5776,8 @@
         {
             "title": "Harbor",
             "hex": "60B932",
-            "source": "https://branding.cncf.io/projects/harbor/"
+            "source": "https://github.com/cncf/artwork/blob/ff2b2b5216e22f001ddd444ca580c484dd10302e/projects/harbor/icon/black/harbor-icon-black.svg",
+            "guidelines": "https://github.com/cncf/artwork/blob/ff2b2b5216e22f001ddd444ca580c484dd10302e/README.md"
         },
```
Comment (adamrusted) [inline _data/simple-icons.json:5781]:
```suggestion
            "guidelines": "https://github.com/cncf/artwork/blob/ff2b2b5216e22f001ddd444ca580c484dd10302e/README.md",
            "license": {
                 "type": "custom",
                 "url": "https://www.linuxfoundation.org/legal/trademark-usage"
             }
        },
```

**Review by adamrusted — COMMENTED** [review]
Icon is spot on, I'd just add the license as we have with other CNCF icons.

**Comment by PratyushJayachandran** [comment]
> Icon is spot on, I'd just add the license as we have with other CNCF icons.

Done

File: _data/simple-icons.json [hunk 2]
```diff
@@ -5776,7 +5776,12 @@
         {
             "title": "Harbor",
             "hex": "60B932",
-            "source": "https://branding.cncf.io/projects/harbor/"
+            "source": "https://github.com/cncf/artwork/blob/ff2b2b5216e22f001ddd444ca580c484dd10302e/projects/harbor/icon/black/harbor-icon-black.svg",
+            "guidelines": "https://github.com/cncf/artwork/blob/ff2b2b5216e22f001ddd444ca580c484dd10302e/README.md",
+            "license": {
+                 "type": "custom",
+                 "url": "https://www.linuxfoundation.org/legal/trademark-usage"
+             }
```
Comment (adamrusted) [inline _data/simple-icons.json:5784]:
```suggestion
            "license": {
                "type": "custom",
                "url": "https://www.linuxfoundation.org/legal/trademark-usage"
            }
```

**Pull request merged**